src/atwho.js

```diff
@@ -131,7 +132,25 @@ define([
                                                        options.property);
                             }
                             return value;
-                        }
+                        },
+                        afterMatchFailed: function(at, $el) {
+                            if (options.useRichText) {
+                                var content = $el.html().trim().replace(/^.*\s/, "");
+                                var isUnknownCaseHashtag = richText.CASE_REF_REGEX.test(content) && content.replace(richText.CASE_REF_REGEX, "");
+                                if (isUnknownCaseHashtag || form.isValidHashtag(content)) {
+                                    options.functionOverrides.insert.call(this, content);
+                                }
+                            }
+
+                            // Gracelessly force focus back to input...otherwise, later code steals
+                            // the focus, or, occasionally, gives focus to a different input.
+                            var $inputor = this.$inputor;
+                            _.delay(function() {
+                                $inputor.focus();
+                            }, 100);
```

Comment:
10ms would likely be fine. I tend to use 100ms delays because it's approximately the longest time that you can wait without a human user perceiving a delay. Some background: https://www.nngroup.com/articles/response-times-3-important-limits/